### DISCUSSION TITLE
Take into account the extrusion "delay"

### DISCUSSION THREAD

**Discussion by rraallvv** [supermerill/SuperSlicer#4408] [Q&A] [answered]
I'm printing with recycled PET filament that has a cavity in it, so I'm over-extruding by a factor of 1.9 set in SuperSlicer. The nozzle has three cavities to ensure high heat transfer from the hotend to the filament. However, I've noticed that the melted filament starts coming out of the nozzle after a noticeable delay once the extruder motor starts moving, sometimes taking more than a second. This delay remains fairly constant throughout the printing process. The effect is most noticeable when printing the support structure, with the supports looking like a bunch of spaghetti instead of a grid. Is it possible to account for this delay in the a slicer settings?
![P1340210+copy](https://github.com/user-attachments/assets/e27fd0dd-5aec-488f-8360-1213d418a29e)
![416dlUUD6sL _SL1200_](https://github.com/user-attachments/assets/f2461f1f-283b-4256-9670-588b4092d82c)

**Comment by EchoAlpha519** [comment, marked as answer]
You might need to play around with the "extra length on restart" setting. It pushes extra filament after a retraction.

**Reply by rraallvv** [reply]
Thanks. This is exactly what I was looking for.

**Comment by supermerill** [comment]
In 2.7, there will be an experimental feature to do a kind of "poor's man pressure compensation" (if you don't have a pressure advance in your firmware). It's not able to create a delay yet, but it's in the pipeline for later (when the 2.7 will be out).
